### PR TITLE
docs: Adjust the update command documentation of the public dashboard endpoint

### PR DESCRIPTION
**Pull request by ZPascal**
**What is this feature?**

Update the documentation and adjust the update part of the public dashboard.

**Why do we need this feature?**

Good documentation is necessary to call the API correctly.

**Who is this feature for?**

For all people who use the public dashboard API.

**Labels and Milestones:**
I'm an external contributor, so, it's not possible to add any related labels or milestones, but I would suggest porting the documentation change back.

backport v10.2.0
no-changelog

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by ZPascal** [comment]
@Eve832 Could you please support me here?

**Comment by ZPascal** [comment]
@Eve832 Could you please have a look?

**Comment by ZPascal** [comment]
The topic is still up to date and needs a review from an approver. :)

**Comment by ZPascal** [comment]
@DanCech Could you please support me in handling the PR?

**Review by DanCech — APPROVED** [review]
LGTM

**Pull request merged**
